tests/test_server.py

```diff
@@ -221,3 +221,31 @@ def dev_status(self):
         assert proxy.attr == 1.23
         assert proxy.attr2 == 3.14
         assert proxy.status() == ")`'-.,_)`'-.,_)`'-.,_"
+
+
+def test_polled_attribute(server_green_mode):
+
+    dct = {'Polling1': 100, 'Polling2': 100000, 'Polling3': 500}
+
+    class TestDevice(Device):
+        green_mode = server_green_mode
+
+        @attribute(polling_period=dct["Polling1"])
+        def Polling1(self):
+            pass
+
+        @attribute(polling_period=dct["Polling2"])
+        def Polling2(self):
+            pass
+
+        @attribute(polling_period=dct["Polling3"])
+        def Polling3(self):
+            pass
```

Comment:
Those attributes are `float`by default, so it'd be nicer if the methods return a float. Also, what about more explicit names, such as `PolledAttributte3`?